setup.py

```diff
@@ -232,6 +232,7 @@ def do_setup():
             'pandas>=0.17.1, <1.0.0',
             'pendulum==1.4.0',
             'psutil>=4.2.0, <5.0.0',
+            'pydruid==0.4.1',
```

Comment:
Let's create an `extra_requires` group for `druid` as most Airflow installs won't require it. It should have been there already but isn't...

Reply:
sounds good. will update the pr :)